### ISSUE TITLE
Imputs from Tensorflow network are not radomized

### ISSUE THREAD

**Issue by scaverod** [scaverod/MalariaDL#1]
The inputs of the neural network made with tensorflow must be random.  In addition, it would be appropriate to separate the dataset into a training and test set.
